examples/README.md

```diff
@@ -2,6 +2,15 @@
 
 The Toit examples are meant to get you started writing code for an ESP32.
 
+## Packages
+
+Some of the examples use other packages, e.g. HTTP. To run the examples you first need to run
+
+```
```

Comment:
Drop $ from the instructions and change it to:

``` sh
build/host/sdk/bin/toitpkg pkg install
```

(adding sh and using the built toitpkg)